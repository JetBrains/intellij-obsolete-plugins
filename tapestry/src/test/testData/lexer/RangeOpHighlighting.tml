    <t:hidden value="44" encoder="${rrrrr?.tt?.meth(55..aa?.bb(), ww..-4, ee.tt..aa, ee.hh()..dd.vv)}"></t:hidden>