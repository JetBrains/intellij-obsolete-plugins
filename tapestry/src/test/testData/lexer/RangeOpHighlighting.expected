XML_WHITE_SPACE ('    ')
XML_START_TAG_START ('<')
XML_TAG_NAME ('t:hidden')
TAG_WHITE_SPACE (' ')
XML_NAME ('value')
XML_EQ ('=')
XML_ATTRIBUTE_VALUE_START_DELIMITER ('"')
XML_ATTRIBUTE_VALUE_TOKEN ('44')
XML_ATTRIBUTE_VALUE_END_DELIMITER ('"')
TAG_WHITE_SPACE (' ')
XML_NAME ('encoder')
XML_EQ ('=')
XML_ATTRIBUTE_VALUE_START_DELIMITER ('"')
${ ('${')
TAP5_EL_IDENTIFIER ('rrrrr')
?. ('?.')
TAP5_EL_IDENTIFIER ('tt')
?. ('?.')
TAP5_EL_IDENTIFIER ('meth')
( ('(')
TAP5_EL_INTEGER ('55')
.. ('..')
TAP5_EL_IDENTIFIER ('aa')
?. ('?.')
TAP5_EL_IDENTIFIER ('bb')
( ('(')
) (')')
, (',')
WHITE_SPACE (' ')
TAP5_EL_IDENTIFIER ('ww')
.. ('..')
TAP5_EL_INTEGER ('-4')
, (',')
WHITE_SPACE (' ')
TAP5_EL_IDENTIFIER ('ee')
. ('.')
TAP5_EL_IDENTIFIER ('tt')
.. ('..')
TAP5_EL_IDENTIFIER ('aa')
, (',')
WHITE_SPACE (' ')
TAP5_EL_IDENTIFIER ('ee')
. ('.')
TAP5_EL_IDENTIFIER ('hh')
( ('(')
) (')')
.. ('..')
TAP5_EL_IDENTIFIER ('dd')
. ('.')
TAP5_EL_IDENTIFIER ('vv')
) (')')
} ('}')
XML_ATTRIBUTE_VALUE_END_DELIMITER ('"')
XML_TAG_END ('>')
XML_END_TAG_START ('</')
XML_TAG_NAME ('t:hidden')
XML_TAG_END ('>')
